src/main/java/net/minecraftforge/common/extensions/IForgeRecipeBookCategory.java

```diff
@@ -0,0 +1,16 @@
+package net.minecraftforge.common.extensions;
+
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.crafting.IRecipe;
+import net.minecraft.item.crafting.IRecipeType;
+import net.minecraftforge.registries.IForgeRegistryEntry;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+public interface IForgeRecipeBookCategory<T extends IRecipeType<?>> extends IForgeRegistryEntry<IForgeRecipeBookCategory<?>> {
```

Comment:
Documentation is needed here.

JavaDoc or otherwise.

What does every method do.
You are working on an API that others use and need to maintain. You are not in a Patch, so be as detailed as you can be with this.

Reply:
Added  JavaDoc